client/src/components/Header/components/nav-links.tsx

```diff
@@ -50,6 +53,49 @@ const mapDispatchToProps = {
   openSignoutModal
 };
 
+interface DonateButtonProps {
+  isUserDonating: boolean | undefined;
+  handleMenuKeyDown: (
+    event: React.KeyboardEvent<HTMLButtonElement | HTMLAnchorElement>
+  ) => void;
```

Comment:
until I can sort the unauthorized api call, this a correction for the type because `Link` can only be a Link

```suggestion
  handleMenuKeyDown: (event: React.KeyboardEvent<HTMLAnchorElement>) => void;
```